model_analyzer/result/sorted_results.py

```diff
@@ -35,53 +40,55 @@ def empty(self):
         """
         Returns
         -------
-        True if this heap has no results
+        True if this list has no results
```

Comment:
refer to the class/object, not to "this list"

Reply:
Fixed